### PR TITLE
Add working sign-in page

### PR DESCRIPTION
**Pull request by lukaszsi**
dactylSignIn

### CODE REVIEW

File: ui/src/pages/SignIn/queries.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { gql } from "@apollo/client";
+
+export const LOGIN = gql`
+  mutation LoginUser($email: String!, $password: String!) {
+    loginUser(input: { email: $email, password: $password }) {
+      email
```
Comment (crypturgus) [inline ui/src/pages/SignIn/queries.ts:6]: I should return a jwt or a user object (only e-mail for now)?
Currently returns header: ` Authorization: Bearer <token>`?

Comment (lukaszsi) [reply ui/src/pages/SignIn/queries.ts:6]: Both I'd say. User will be useful for some general user data context and token to be stored and provided later in queries.

File: ui/src/pages/SignIn/queries.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export const LOGIN = gql`
+  mutation LoginUser($email: String!, $password: String!) {
+    loginUser(input: { email: $email, password: $password }) {
+      user {
+        email
+      }
+      token
+    }
+  }
+`;
```
Comment (crypturgus) [inline ui/src/pages/SignIn/queries.ts:12]: missing newline?
perhaps we could add a front-end linter?

Comment (lukaszsi) [reply ui/src/pages/SignIn/queries.ts:12]: Sure, it is somewhere on a list 👍

**Pull request merged**